### PR TITLE
adding DeVaucouleurs & Exponential profile

### PR DESCRIPTION
**Pull request by b-remy**
Here is a small pull request adding DeVaucouleurs light profile so that we don't have to deal with sersic index and its normalizing factor, as suggested by @aguinot here: https://github.com/b-remy/gems/issues/1#issuecomment-934304200.

### CODE REVIEW

**Comment by b-remy** [comment]
And also Exponential light profiles.

File: galflow/python/lightprofiles.py [hunk 1]
```diff
@@ -89,6 +89,80 @@ def gaussian(fwhm=None, half_light_radius=None, sigma=None, flux=None, scale=1.,
...
+    nx: `int`, width of the stamp
+    ny: `int`, height of the stamp
+
+  Returns:
+    `Tensor` of shape [batch_size, nx, ny] of the centered profile
+
+  Example:
+    >>> sersic(n=[2.], scale_radius=[5.], flux=[40.], nx=55)
+  """
+  with tf.name_scope(name or "sersic"):
```
Comment (EiffL) [inline galflow/python/lightprofiles.py]:
```suggestion
  with tf.name_scope(name or "exponential"):
```

File: galflow/python/lightprofiles.py [hunk 2]
```diff
@@ -89,6 +89,80 @@ def gaussian(fwhm=None, half_light_radius=None, sigma=None, flux=None, scale=1.,
...
+    flux: List of `float`, flux (in photons/cm^2/s) of the profile. [default: 1]
+    scale: `float`, the pixel scale to use for the drawn image
+    nx: `int`, width of the stamp
+    ny: `int`, height of the stamp
+
+  Returns:
+    `Tensor` of shape [batch_size, nx, ny] of the centered profile
+
+  Example:
+    >>> sersic(n=[2.], scale_radius=[5.], flux=[40.], nx=55)
```
Comment (EiffL) [inline galflow/python/lightprofiles.py]:
```suggestion
    >>> exponential(scale_radius=[5.], flux=[40.], nx=55)
```

**Review by EiffL — CHANGES_REQUESTED** [review]
Looking good, only two small corrections that I could find

**Review by EiffL — APPROVED** [review]
LGTM

**Pull request merged**